tests/playground.html

```diff
@@ -436,6 +445,8 @@
     <br>
     <input type="button" value="To JavaScript" onclick="toCode('JavaScript')">
     &nbsp;
+    <input type="button" value="To PHP" onclick="toCode('PHP')">
```

Comment:
In all other cases the order is JS/Python/PHP/Dart.  Flip PHP and Python here to be consistent.